### PR TITLE
Upgrade to 2.13.0 (and the new collections)

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

**Comment by julienrf** [comment]
Only the `scala-parser-combinatorsJVM` module compiles. We still lack a 2.13 compatible Scala.js release.

**Comment by gourlaysama** [comment]
Moved to #152.

File: shared/src/main/scala-2.13/scala/util/parsing/input/ScalaVersionSpecificPagedSeq.scala [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package scala.util.parsing.input
+
+private[input] trait ScalaVersionSpecificPagedSeq[T] { self: PagedSeq[T] =>
+  // Members declared in scala.collection.Seq
+  override def iterableFactory: collection.SeqFactory[collection.IndexedSeq] = collection.IndexedSeq
```
Comment (MasseGuillaume) [inline shared/src/main/scala-2.13/scala/util/parsing/input/ScalaVersionSpecificPagedSeq.scala]: @julienrf why is this required?

Comment (julienrf) [reply shared/src/main/scala-2.13/scala/util/parsing/input/ScalaVersionSpecificPagedSeq.scala]: Maybe this is not anymore required, actually. Because we override it with the same body as the inherited one…
